backend/services/status.js

```diff
@@ -3,11 +3,19 @@ const statuses = {
 };
 
 class Status {
-  constructor(names) {
+  constructor(crawlersConfig) {
```

Comment:
This class is responsible for checking status. It is not responsible for selecting for whom this status should be checked. 
Move the filtering logic out from this class.

Reply:
Moved out to helpers

Reply:
It depends on if the status services is operational or lib services. If it's an operational service then filtering can be here. Otherwise we need to filter it in `index.js` and move `Status` service to `libs`.

Reply:
I've left Status where it is.
Moving this class is out of the scope. And I guess we need a kind of refactoring plan before doing it.

Reply:
Seems to me as a simple update that doesn't require many efforts as it's the first service in the services folder. It's better to do it now rather than later.

Reply:
Despite the fact that the folder is called `libs`, the inner file `utils` contains domain-related functions. Also, it's not clear whether should we have `services`  folder inside `libs` or just `status.js`

Reply:
> It depends on if the status service is an operational or lib infrastructural service. If it's an operational service then filtering can be here.

We are following SOLID. If you put filtering logic inside the class, you break `S` and `O`.

> Seems to me as a simple update that doesn't require many efforts as it's the first service in the services folder. It's better to do it now rather than later.

@MRamanenkau, I agree that currently in `libs` exists a class with domain-related logic at the same time I agree with @Dmitriy1984Molchanov that `status.js` should be there. Let's move `status.js` to the `libs` folder in this PR. This refactoring suits this PR.

Reply:
@AndreiNavoichyk, If we decide to distillate its logic to reusable infrastructural service then it shouldn't know anything about application operations. 
An `operational` (`application`)  service may accumulate multiple logical operations (sending email, saving to db, filtering enabled crawlers) so it's not possible to limit its logic strictly. Its single responsibility would be to process all operations related to status logic operations and filtering could be included.

Reply:
`Status` is moved to `libs`